src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsCardView/DeleteCircle.swift

```diff
@@ -0,0 +1,44 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+
+/// A red circle to indicate a delete action
+@IBDesignable
+class DeleteCircle: UIView {
```

Comment:
Is there a reason why we are drawing this manually instead of using an image?

Reply:
Because it's cool!?

Reply:
Hm, well, I guess an image would be less(/no) code, and less code would be easier to understand and maintain, which I think would be cooler 😄